### DISCUSSION TITLE
JSON Responses:  camelCase field names or snake_case

### DISCUSSION THREAD

**Discussion by steverawlins-zebra** [zalopay-oss/jmeter-grpc-request#110] [Q&A]
v1.2.1 returns JSON fields such as "file_name_here"
but a few versions ago (1.1?)  same came back as "fileNameHere"

How do you determine the response?  Is there a toggle option somewhere?

**Comment by xGaika** [comment]
I have same problem

**Reply by steverawlins-zebra** [reply]
I believe the protobuf-style field_name_with_underscores is the latest and so, if you have not already, you should upgrade the plugin to v1.2.1

I had a separate issue, now Closed, here:
https://github.com/zalopay-oss/jmeter-grpc-request/issues/100
